cdap-cli/src/main/java/co/cask/cdap/cli/command/CallServiceCommand.java

```diff
@@ -114,9 +127,10 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
 
   @Override
   public String getPattern() {
-    return String.format("call service <%s> <%s> <%s> [headers <%s>] [body <%s>]",
+    return String.format("call service <%s> <%s> <%s> [headers <%s>] [body <%s>] [body:file <%s>]",
```

Comment:
Will this allow me to specify both a string and a file? Or will the CLI auto-complete to only one of them?